packages/itwinui-css/src/anchor/anchor.scss

```diff
@@ -50,13 +49,12 @@
 @mixin iui-anchor-status($status: primary) {
   @include iui-focus($color: var(--iui-color-foreground-#{$status}), $offset: 1px);
   color: var(--iui-color-foreground-#{$status});
-  -webkit-tap-highlight-color: rgba(var(--iui-color-foreground-#{$status}-rgb), var(--iui-opacity-5));
   @if $status != primary {
```

Comment:
why highlight color was removed?

Reply:
because we don't/didn't have the transparency variables (and because jon was ok with removing it). i think i might add it back though

Reply:
added back